lib/components/dropdown.vue

```diff
@@ -43,15 +43,6 @@
     </div>
 </template>
 
-<style scoped>
-    .dropdown-item:focus,
-    .dropdown-item:hover,
-    .dropdown-header:focus {
-        background-color: #eaeaea;
```

Comment:
It was glitching on mouse-down state (overriding blue background) also `scoped` is not a good idea.  But got the point. will try to revert it back with fixes.